bitrise.yml

```diff
@@ -0,0 +1,44 @@
+format_version: "8"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test:
+    envs:
+    - APK_DOWNLOAD_URL: https://raw.githubusercontent.com/bitrise-io/sample-artifacts/master/apks/app-armeabi-v7a-release.apk
+    - APK_PATH: app-debug.apk
+    - MAPPING_DOWNLOAD_URL: https://raw.githubusercontent.com/bitrise-io/sample-artifacts/master/mappings/app-armeabi-v7a-release.txt
+    - MAPPING_PATH: mapping.txt
+    - APPCENTER_APP_NAME: steps-appcenter-deploy-android
+    - APPCENTER_APP_OWNER: app-center-bot
+    steps:
+    steps:
+    - go-list:
+    - golint:
+    - errcheck:
+    - go-test:
+    - script:
+        title: Clean _tmp
+        inputs:
+          - content: rm -rf ./_tmp
+    - change-workdir:
+        title: Change workdir to _tmp
+        run_if: true
+        inputs:
+          - path: ./_tmp
+    - script:
+        title: Download testing resources
+        inputs:
+          - content: |-
+              #!/bin/bash
+              set -ex
+              curl $APK_DOWNLOAD_URL > $APK_PATH
+              curl $MAPPING_DOWNLOAD_URL > $MAPPING_PATH
+    - path::./:
+        inputs:
+          - apk_path: $APK_PATH
+          - api_token: $API_TOKEN
+          - owner_name: $APPCENTER_APP_OWNER
+          - app_name: $APPCENTER_APP_NAME
+          - distribution_group: Collaborators
+          - mapping_path: $MAPPING_PATH
+          - release_notes: Bitrise step test
```

Comment:
afther this step check please have a script step and validate all the exported output envs values, what you can exactly, what you cant for emptyness

Reply:
no output checks yet